src/Core/Content/Mail/Service/MailSender.php

```diff
@@ -11,22 +11,23 @@
 use Shopware\Core\Framework\Util\Hasher;
 use Shopware\Core\System\SystemConfig\SystemConfigService;
 use Symfony\Component\Mailer\Envelope;
-use Symfony\Component\Mailer\Transport\TransportInterface;
+use Symfony\Component\Mailer\MailerInterface;
 use Symfony\Component\Messenger\MessageBusInterface;
 use Symfony\Component\Mime\Email;
 
 #[Package('after-sales')]
 class MailSender extends AbstractMailSender
 {
     public const DISABLE_MAIL_DELIVERY = 'core.mailerSettings.disableDelivery';
+    private const MESSAGE_SIZE_LIMIT = 1024 * 256;
```

Comment:
is there a specific reason why we use this number? maybe there could be some explanation or the constant could have another name accordingly